tachyon/zk/plonk/vanishing/vanishing_argument.h

```diff
@@ -75,8 +80,32 @@ class VanishingArgument {
     return evaluator;
   }
 
-  // TODO(chokobole): Implement EvaluateH. See
-  // [evaluate_h](https://github.com/kroma-network/halo2/blob/7d0a36990452c8e7ebd600de258420781a9b7917/halo2_proofs/src/plonk/evaluation.rs#L279-L583).
+  const GraphEvaluator<F>& custom_gates() const { return custom_gates_; }
+  const std::vector<GraphEvaluator<F>> lookups() const { return lookups_; }
+
+  template <typename PCSTy, typename Poly = typename PCSTy::Poly,
+            typename ExtendedEvals = typename PCSTy::ExtendedEvals>
+  ExtendedEvals BuildExtendedCircuitColumn(
```

Comment:
```suggestion
  template <typename PCSTy, typename Poly>
  typename PCSTy::ExtendedEvals BuildExtendedCircuitColumn(
```

Reply:
This is a meaningful comment, but currently, we are not strictly limiting the relationship between template arguments. We will have to discuss this part again later.

Reply:
Left an issue https://github.com/kroma-network/tachyon/issues/210.